habitTracker.jakubmikolajewski/HabitTrackerLibrary/DatabaseQueries.cs

```diff
@@ -0,0 +1,434 @@
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"Exception occurred during creating database file.\nDetails: {ex.Message}");
+                }
+                GenerateRandomDatabase();
+            }
+        }
+        private void GenerateRandomDatabase()
+        {
+            RandomDateTime date = new RandomDateTime();
+            List<string> habits = new List<string>() { "Running", "Swimming", "Cycling", "Walking" };
+
+            using (connection = TryOpenConnection())
+            {
+                try
+                {
+                    foreach (string habit in habits)
+                    {
+                        command = connection.CreateCommand();
+                        command.CommandText = $"create table {habit} (occurrenceId int, occurrenceDate text, distance int)";
+                        command.ExecuteNonQuery();
+                        for (int i = 0; i < 100; i++)
+                        {
+                            string randomDate = date.Next().ToString("yyyy-MM-dd HH:mm");
+                            int randomDistance = Random.Shared.Next(1, 20);
+                            using (command = connection.CreateCommand())
+                            {
+                                command.CommandText = $"insert into {habit} (occurrenceId, occurrenceDate, distance) values ({i}, '{randomDate}', {randomDistance})";
+                                command.ExecuteNonQuery();
+                            }
+                        }
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"Exception occurred during generating database.\nDetails: {ex.Message}");
+                }
+                finally
+                {
+                    connection.Close();
+                }
+            }
+        }
+        public void GetCurrentTables()
+        {
+            currentTables.Clear();
+            using (connection = TryOpenConnection())
+            {
+                try
+                {
+                    using (command = connection.CreateCommand())
+                    {
+                        command.CommandText = "select name from sqlite_master";
+                        using (SQLiteDataReader reader = command.ExecuteReader())
+                            while (reader.Read())
+                            {
+                                currentTables.Add(reader.GetString(0));
+                            }
+                    }
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine($"Exception occurred during getting table information.\nDetails: {ex.Message}");
+                }
+                finally
+                {
+                    connection.Close();
+                }
+            }
+        }
+        public void CreateHabitQuery(string habitName, string chosenMeasurement)
```

Comment:
I like this option.  This would be dangerous in a live program, but you will learn to deal with that in the future.  Good Job.